### PR TITLE
Bg language part one

### PR DESCRIPTION
**Pull request by delphipro**
## Purpose

I fixed several files with Bulgarian translation

### CODE REVIEW

File: base/applications/mspaint/lang/bg-BG.rc [hunk 1]
```diff
@@ -206,13 +206,13 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_PROGRAMNAME "Рисуване"
-    IDS_WINDOWTITLE "%s - рисуване"
-    IDS_INFOTITLE "Рисувач (Paint) за РеактОС"
+    IDS_PROGRAMNAME "Paint"
+    IDS_WINDOWTITLE "%s - paint"
```
Comment (HBelusca) [inline base/applications/mspaint/lang/bg-BG.rc:210]:
```suggestion
    IDS_WINDOWTITLE "%s - Paint"
```

File: dll/cpl/inetcpl/lang/bg-BG.rc [hunk 2]
```diff
@@ -0,0 +1,134 @@
+/*
+ * English resources for the Internet control panel applet
```
Comment (HBelusca) [inline dll/cpl/inetcpl/lang/bg-BG.rc:2]:
```suggestion
 * Bulgarian resources for the Internet control panel applet
```

File: dll/win32/crypt32/crypt32.rc [hunk 3]
```diff
@@ -31,7 +31,9 @@
 
 /* UTF-8 */
 #pragma code_page(65001)
-
+#ifdef LANGUAGE_BG_BG
```
Comment (HBelusca) [inline dll/win32/crypt32/crypt32.rc:34]:
```suggestion

#ifdef LANGUAGE_BG_BG
```

File: dll/win32/cryptui/cryptui.rc [hunk 4]
```diff
@@ -46,6 +46,10 @@ IDB_CERT_HEADER BITMAP "res/certheader.bmp"
 /* UTF-8 */
 #pragma code_page(65001)
 
+#ifdef LANGUAGE_BG_BG
+#include "lang/cryptui_Bg.rc"
+#endif
+
```
Comment (HBelusca) [inline dll/win32/cryptui/cryptui.rc:52]:
```suggestion
```

File: dll/win32/cryptui/cryptui.rc [hunk 5]
```diff
@@ -46,6 +46,10 @@ IDB_CERT_HEADER BITMAP "res/certheader.bmp"
 /* UTF-8 */
 #pragma code_page(65001)
 
+#ifdef LANGUAGE_BG_BG
+#include "lang/cryptui_Bg.rc"
```
Comment (HBelusca) [inline dll/win32/cryptui/cryptui.rc:50]:
```suggestion
    #include "lang/cryptui_Bg.rc"
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 6]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
+    AUTOCHECKBOX "Покажи секундите", IDC_TASKBARPROP_SECONDS, 120, 153, 80, 10, WS_DISABLED
+    LTEXT "Можете да поддържате лентата подредена, като скривате приложенията, върху които скоро не сте стартирали.", IDC_STATIC, 13, 169, 223, 16, WS_DISABLED
+    AUTOCHECKBOX "Скриване на неактивните иконки", IDC_TASKBARPROP_HIDEICONS, 13, 191, 125, 10
+    PUSHBUTTON "Персонализиране...", IDC_TASKBARPROP_ICONCUST, 168, 191, 70, 14
 END
 
 IDD_TASKBARPROP_STARTMENU DIALOGEX 0, 0, 252, 218
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Пусков изборник"
+CAPTION "Start menu"
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:85]: Left untranslated?

File: base/shell/explorer/lang/bg-BG.rc [hunk 7]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
         MENUITEM "От&гнездване на компютъра", IDM_UNDOCKCOMPUTER, MFT_STRING, MFS_ENABLED
         MENUITEM "&Изключване...", IDM_SHUTDOWN, MFT_STRING, MFS_ENABLED
     END
 END
 
-IDD_TASKBARPROP_TASKBAR DIALOGEX 0, 0, 252, 218
+IDD_TASKBARPROP_TASKBAR DIALOGEX 0, 60, 252, 218
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Taskbar"
+CAPTION "Taksbar"
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:64]: Left untranslated? (and there's a typo now)

File: base/shell/explorer/lang/bg-BG.rc [hunk 8]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
         MENUITEM "&Изход %s...", IDM_LOGOFF, MFT_STRING, MFS_ENABLED
-        MENUITEM "&Откачване...", IDM_DISCONNECT, MFT_STRING, MFS_ENABLED
+        MENUITEM "&Откачане...", IDM_DISCONNECT, MFT_STRING, MFS_ENABLED
         MENUITEM "От&гнездване на компютъра", IDM_UNDOCKCOMPUTER, MFT_STRING, MFS_ENABLED
         MENUITEM "&Изключване...", IDM_SHUTDOWN, MFT_STRING, MFS_ENABLED
     END
 END
 
-IDD_TASKBARPROP_TASKBAR DIALOGEX 0, 0, 252, 218
+IDD_TASKBARPROP_TASKBAR DIALOGEX 0, 60, 252, 218
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:62]: OK that change is weird.

**Review by HBelusca — CHANGES_REQUESTED** [review]
Some last points to investigate.
Please also ensure that, in the new files, encoding is UTF-8 and newlines are in the CR-LF format.

File: base/shell/explorer/lang/bg-BG.rc (hunk 6 above)
Comment (delphipro) [reply base/shell/explorer/lang/bg-BG.rc:85]: Correct is "Старт меню"
image

but after click on "Старт меню". 
image

so I returned the old translation

Comment (HBelusca) [reply base/shell/explorer/lang/bg-BG.rc:85]: huh lol
Looks like a bug.

File: dll/win32/msgina/lang/bg-BG.rc [hunk 9]
```diff
@@ -142,18 +142,18 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUPWINDOW | WS_CAPTION
 CAPTION "Изключване на РеактОС"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
+    CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Статично", SS_BITMAP, 0, 0, 275, 54
```
Comment (binarymaster) [inline dll/win32/msgina/lang/bg-BG.rc:145]: This "Static" is non-localizable, it should not be translated.

**Comment by binarymaster** [comment]
@HBelusca the PR author seems to be unavailable, you can probably assign it to yourself, or I can help as well.

Note that his GitHub name is cyrillic, while we may prefer latin characters for the name -- so a manual merge will be needed in this case.

image

**Comment by HBelusca** [comment]
Yes I'm going to do the necessary adjustments.

File: base/shell/explorer/lang/bg-BG.rc (hunk 7 above)
Comment (winterhell) [reply base/shell/explorer/lang/bg-BG.rc]: I've seen Taskbar in literature as "Лента с инструменти"

File: dll/cpl/inetcpl/inetcpl.rc [hunk 10]
```diff
@@ -27,6 +27,9 @@
  /* UTF-8 */
  #pragma code_page(65001)
```
Comment (HBelusca) [inline dll/cpl/inetcpl/inetcpl.rc:28]:
```suggestion
/* UTF-8 */
#pragma code_page(65001)
```

File: dll/win32/msgina/lang/bg-BG.rc [hunk 11]
```diff
@@ -12,7 +12,7 @@ END
 
 IDD_WELCOME DIALOGEX 0, 0, 275, 78
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_BORDER | WS_CAPTION | WS_DLGFRAME | WS_POPUP
-CAPTION "РеактОС ви приветства"
+CAPTION "Добре дошли"
```
Comment (HBelusca) [inline dll/win32/msgina/lang/bg-BG.rc:15]: English translation and others have the equivalent of "Welcome to ReactOS"

File: dll/win32/msgina/lang/bg-BG.rc (hunk 9 above)
Comment (HBelusca) [inline dll/win32/msgina/lang/bg-BG.rc:145]:
```suggestion
    CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
```

File: base/shell/explorer/lang/bg-BG.rc (hunk 8 above)
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:62]:
```suggestion
IDD_TASKBARPROP_TASKBAR DIALOGEX 0, 0, 252, 218
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 12]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
 FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
-CONTROL "", IDC_TASKBARPROP_STARTMENU_BITMAP, "Static", SS_BITMAP | SS_SUNKEN,7, 7, 183, 108
-    AUTORADIOBUTTON "&Пусков изборник", IDC_TASKBARPROP_STARTMENU, 7, 135, 105, 10
-    LTEXT "Този изборник ви дава лесне достъп до папките ви, любимите ви приложения и търсенето.", IDC_TASKBARPROP_STARTMENUMODERNTEXT, 20, 145, 150, 24
-    PUSHBUTTON "На&гласяване...", IDC_TASKBARPROP_STARTMENUCUST, 192, 132, 53, 14
-    AUTORADIOBUTTON "Класически пусков из&борник", IDC_TASKBARPROP_STARTMENUCLASSIC, 7, 175, 105, 10
-    LTEXT "Този изглед на изборника ви осигурява класически облик и начин на работа.", IDC_TASKBARPROP_STARTMENUCLASSICTEXT, 20, 185, 150, 24
-    PUSHBUTTON "На&гласяване...", IDC_TASKBARPROP_STARTMENUCLASSICCUST, 192, 172, 53, 14
+CONTROL "", IDC_TASKBARPROP_STARTMENU_BITMAP, "Статично", SS_BITMAP | SS_SUNKEN,7, 7, 183, 108
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:88]:
```suggestion
    CONTROL "", IDC_TASKBARPROP_STARTMENU_BITMAP, "Static", SS_BITMAP | SS_SUNKEN, 7, 7, 183, 108
```

File: dll/win32/msgina/lang/bg-BG.rc (hunk 11 above)
Comment (HBelusca) [inline dll/win32/msgina/lang/bg-BG.rc:15]:
```suggestion
CAPTION "Добре Дошли в ReactOS"
```

File: base/applications/calc/lang/bg-BG.rc [hunk 13]
```diff
@@ -223,7 +223,7 @@ END
 
 IDD_DIALOG_CONVERSION DIALOGEX 0, 0, 320, 130
 STYLE DS_SHELLFONT | WS_MINIMIZEBOX | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Изчислителят на РеактОС"
+CAPTION "Калкулатор на ReactOS"
```
Comment (HBelusca) [inline base/applications/calc/lang/bg-BG.rc:226]:
```suggestion
CAPTION "Калкулатор за ReactOS"
```

File: base/shell/explorer/lang/bg-BG.rc (hunk 7 above)
Comment (HBelusca) [reply base/shell/explorer/lang/bg-BG.rc]: ...or "Лентата на задачите" ?
https://support.microsoft.com/bg-bg/windows/%D0%BA%D0%B0%D0%BA-%D0%B4%D0%B0-%D0%B8%D0%B7%D0%BF%D0%BE%D0%BB%D0%B7%D0%B2%D0%B0%D1%82%D0%B5-%D0%BB%D0%B5%D0%BD%D1%82%D0%B0%D1%82%D0%B0-%D0%BD%D0%B0-%D0%B7%D0%B0%D0%B4%D0%B0%D1%87%D0%B8%D1%82%D0%B5-%D0%B2-windows-11-0657a50f-0cc7-dbfd-ae6b-05020b195b07

File: base/shell/explorer/lang/bg-BG.rc [hunk 14]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
-    AUTORADIOBUTTON  "&Large icons", IDC_CHOOSELARGE, 43, 25, 53, 10
-    AUTORADIOBUTTON  "&Small icons", IDC_CHOOSESMALL, 177, 25, 51, 10, NOT WS_TABSTOP
-    GROUPBOX         "Programs ", IDC_STATIC, 7, 49, 238, 70
-    LTEXT            "The Start menu contains shortcuts to the programs you use most often.  Clearing the list of shortcuts does not delete the programs.", IDC_STATIC, 14, 63, 224, 20
-    LTEXT            "&Number of programs on Start menu:", IDC_STATIC, 14, 86, 112, 8
+    AUTORADIOBUTTON  "&Големи иконки", IDC_CHOOSELARGE, 43, 25, 53, 10
+    AUTORADIOBUTTON  "&Малки иконки", IDC_CHOOSESMALL, 177, 25, 51, 10, NOT WS_TABSTOP
+    GROUPBOX         "Програми ", IDC_STATIC, 7, 49, 238, 70
+    LTEXT            "Менюто Старт съдържа преки пътища към програмите, които използвате често. Изтриването на преките пътища на изтрива програмите.", IDC_STATIC, 14, 63, 224, 20
+    LTEXT            "Програми в менюто Старт:", IDC_STATIC, 14, 86, 112, 8
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:174]:
```suggestion
    LTEXT            "&Програми в менюто Старт:", IDC_STATIC, 14, 86, 112, 8
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 15]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
     EDITTEXT         IDC_NUMBEROFPROGRAMS, 133, 84, 27, 12, ES_LEFT | ES_RIGHT | ES_NUMBER
     CONTROL          "", IDC_NUMBERUPDOWN, "msctls_updown32", WS_BORDER | 0x000000B6, 160, 84, 11, 12
-    PUSHBUTTON       "&Clear List", IDC_CLEARLIST, 182, 98, 57, 14
-    GROUPBOX         "Show on Start menu ", IDC_STATIC, 7, 126, 238, 63
-    AUTOCHECKBOX     "&Internet:", IDC_SHOWINTERNET, 14, 140, 42, 10
+    PUSHBUTTON       "&Изчисти списъка", IDC_CLEARLIST, 182, 98, 57, 14
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:177]:
```suggestion
    PUSHBUTTON       "Изчисти &списъка", IDC_CLEARLIST, 182, 98, 57, 14
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 16]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
+CAPTION "Предупреждение на името на файла"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    DEFPUSHBUTTON    "&Ignore", IDIGNORE, 215, 60, 50, 14
-    PUSHBUTTON       "&Rename", IDC_FILE_RENAME, 159, 60, 50, 14
+    DEFPUSHBUTTON    "&Игнорирай", IDIGNORE, 215, 60, 50, 14
+    PUSHBUTTON       "&Преименувай", IDC_FILE_RENAME, 159, 60, 50, 14
     LTEXT            "", IDC_STATIC, 7, 51, 258, 1, NOT WS_GROUP
-    AUTOCHECKBOX     "&Don't perform this check at startup", IDC_NOTCHECK, 7, 62, 139, 10
+    AUTOCHECKBOX     "Не прави тази проверка при стартиране", IDC_NOTCHECK, 7, 62, 139, 10
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:141]:
```suggestion
    AUTOCHECKBOX     "&Не прави тази проверка при стартиране", IDC_NOTCHECK, 7, 62, 139, 10
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 17]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
-    GROUPBOX         "Start menu", IDC_STATIC, 8, 7, 218, 114
+    GROUPBOX         "Старт меню", IDC_STATIC, 8, 7, 218, 114
     ICON             IDI_STARTMENU2, IDC_STATIC, 19, 17, 20, 20
-    LTEXT            "You can customize your Start menu by adding or removing items.", IDC_STATIC, 48, 17, 108, 39
-    PUSHBUTTON       "A&dd...", IDC_CLASSICSTART_ADD, 162, 18, 53, 14
-    PUSHBUTTON       "&Remove...", IDC_CLASSICSTART_REMOVE, 162, 33, 53, 14
-    PUSHBUTTON       "Ad&vanced", IDC_CLASSICSTART_ADVANCED, 162, 49, 53, 14
-    PUSHBUTTON       "&Sort", IDC_CLASSICSTART_SORT, 162, 65, 53, 14
+    LTEXT            "Можете да персонализирате Старт менюто като добавяте и премахвате елементи.", IDC_STATIC, 48, 17, 108, 39
+    PUSHBUTTON       "Добави...", IDC_CLASSICSTART_ADD, 162, 18, 53, 14
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:120]:
```suggestion
    PUSHBUTTON       "&Добави...", IDC_CLASSICSTART_ADD, 162, 18, 53, 14
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 18]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
+    PUSHBUTTON       "Премахни...", IDC_CLASSICSTART_REMOVE, 162, 33, 53, 14
+    PUSHBUTTON       "Разширени", IDC_CLASSICSTART_ADVANCED, 162, 49, 53, 14
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:122]:
```suggestion
    PUSHBUTTON       "&Премахни...", IDC_CLASSICSTART_REMOVE, 162, 33, 53, 14
    PUSHBUTTON       "&Разширени", IDC_CLASSICSTART_ADVANCED, 162, 49, 53, 14
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 19]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
-    AUTORADIOBUTTON "&Пусков изборник", IDC_TASKBARPROP_STARTMENU, 7, 135, 105, 10
-    LTEXT "Този изборник ви дава лесне достъп до папките ви, любимите ви приложения и търсенето.", IDC_TASKBARPROP_STARTMENUMODERNTEXT, 20, 145, 150, 24
-    PUSHBUTTON "На&гласяване...", IDC_TASKBARPROP_STARTMENUCUST, 192, 132, 53, 14
-    AUTORADIOBUTTON "Класически пусков из&борник", IDC_TASKBARPROP_STARTMENUCLASSIC, 7, 175, 105, 10
-    LTEXT "Този изглед на изборника ви осигурява класически облик и начин на работа.", IDC_TASKBARPROP_STARTMENUCLASSICTEXT, 20, 185, 150, 24
-    PUSHBUTTON "На&гласяване...", IDC_TASKBARPROP_STARTMENUCLASSICCUST, 192, 172, 53, 14
+    CONTROL "", IDC_TASKBARPROP_STARTMENU_BITMAP, "Static", SS_BITMAP | SS_SUNKEN, 7, 7, 183, 108
+    AUTORADIOBUTTON "&Старт меню", IDC_TASKBARPROP_STARTMENU, 7, 135, 105, 10
+    LTEXT "Тази лента със задачите ви дава лесен достъп до папките ви, приложенията и търсенето.", IDC_TASKBARPROP_STARTMENUMODERNTEXT, 20, 145, 150, 24
+    PUSHBUTTON "Персонализиране...", IDC_TASKBARPROP_STARTMENUCUST, 192, 132, 53, 14
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:91]:
```suggestion
    PUSHBUTTON "&Персонализиране...", IDC_TASKBARPROP_STARTMENUCUST, 192, 132, 53, 14
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 20]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
+    AUTORADIOBUTTON "Класическо Старт меню", IDC_TASKBARPROP_STARTMENUCLASSIC, 7, 175, 105, 10
+    LTEXT "Този изглед на лентата със задачите ви осигурява класически облик и начин на работа.", IDC_TASKBARPROP_STARTMENUCLASSICTEXT, 20, 185, 150, 24
+    PUSHBUTTON "Персонализиране...", IDC_TASKBARPROP_STARTMENUCLASSICCUST, 192, 172, 53, 14
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:94]:
```suggestion
    PUSHBUTTON "&Персонализиране...", IDC_TASKBARPROP_STARTMENUCLASSICCUST, 192, 172, 53, 14
```

File: base/shell/explorer/lang/bg-BG.rc [hunk 21]
```diff
@@ -40,159 +40,159 @@ BEGIN
...
+        MENUITEM "&Откачане...", IDM_DISCONNECT, MFT_STRING, MFS_ENABLED
         MENUITEM "От&гнездване на компютъра", IDM_UNDOCKCOMPUTER, MFT_STRING, MFS_ENABLED
         MENUITEM "&Изключване...", IDM_SHUTDOWN, MFT_STRING, MFS_ENABLED
     END
 END
 
 IDD_TASKBARPROP_TASKBAR DIALOGEX 0, 0, 252, 218
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Taskbar"
+CAPTION "Лента със задачи"
```
Comment (HBelusca) [inline base/shell/explorer/lang/bg-BG.rc:64]:
```suggestion
CAPTION "Лентата със задачите"
```

**Comment by HBelusca** [comment]
Translation manually committed in master :)

**Pull request closed without merging**